src/components/Questions/Questions.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import QuestionImage from './../../assets/icons/QuestionImages.svg';
+import Answers from './../Answers/Answers';
+function Questions() {
+  return (
+    <div className="flex flex-col bg-darkgray w-full justify-center items-center">
+      <div className="imgWrapper flex ">
+        <img src={QuestionImage} alt="question" className="flex" />
```

Comment:
you should get this from the question prop
```suggestion
        <img src={question.image} alt="question" className="flex" />
```